_scripts/update.py

```diff
@@ -150,10 +157,14 @@ def write_commit_msg(args, new_files: int, updated_files: int):
 
 
 if __name__ == "__main__":
+    config_file = join("_configs", "config.json")
+    with open(config_file) as f:
+        cgf_automation = json.load(f)
+
     prog = "python -m _scripts.update"
     parser = ArgumentParser(
         prog=prog,
-        description=description,
+        description=__doc__.replace("HEAD", cgf_automation["head"]),
```

Comment:
Is this what you had in mind?

> Would probably be best to make this (and the one for prune.py) an actual docstring, and then to apply the formatting to the __doc__ value's literal {HEAD} to generate the CLI description.

Reply:
This works and is fine; what I specifically had in mind was keeping the `{HEAD}` and then using here:
```suggestion
        description=__doc__.format(HEAD=cgf_automation["head"]),
```